src/styles/blocks/card.scss

```diff
@@ -0,0 +1,63 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  padding-top: 32px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title-wrapper {
+    margin-top: 40px;
+    text-align: left;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-tertiary;
```

Comment:
Ensure that the variable `$color-tertiary` is defined in your SCSS files to avoid compilation errors.